code/__PATH_COMPATIBILITY/path_compatibility.dm

```diff
@@ -1,28 +0,0 @@
-/*
```

Comment:
He's keeping the folder+file, I told him to, He's just taking his time to do it.